services/multitenant/tenantstats.go

```diff
@@ -18,7 +18,7 @@ var (
 	pkgLogger logger.LoggerI
 )
 
-type MultitenantStatsT struct {
+type StatsT struct {
```

Comment:
More idiomatic and readable like this :) 

I would go a step further and 
```suggestion
	t := Stats{}

Reply:
+1 to that :slightly_smiling_face:

Reply:
It means that there is no need for this naming convention of having the types end with a capital `T` e.g. `StatsT`. You can just call it `Stats` which is way more idiomatic in Golang. If it's a type or an interface is already determined by, well, its type :slightly_smiling_face: